packages/terra-scrollable-container/src/ScrollableContainer.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './ScrollableContainer.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The child content to present in the scrollable region.
+   */
+  children: PropTypes.node,
+  /**
+   * Function callback for the ref of the scrollable container.
+   */
+  refCallback: PropTypes.func,
```

Comment:
Instead of first classing this prop, how about we just attach ref to customProps and allow it to go through that way?

Reply:
React does not allow 'ref' for be passed through as a prop. It functions similar to 'key' in that it is a reserved value. React will throw warning at you saying the value is inaccessible.  Also as a functional component ref is not mapped in the standard way, it can only be directly attached to a primitive.